rustls/src/tls13/mod.rs

```diff
@@ -215,3 +216,24 @@ pub(crate) fn is_sigscheme_supported_in_tls13(sigscheme: &SignatureScheme) -> bo
             | SignatureScheme::ED25519
     )
 }
+
+/// Constructs the signature message specified in section 4.4.3 of RFC8446.
+pub(crate) fn construct_tls13_client_verify_message(handshake_hash: &Digest) -> Vec<u8> {
```

Comment:
this is another place where I wonder whether it makes sense to drop the `tls13` portion of the name and let module-qualified references to the `fn` disambiguate from other protocol versions. 

I think the existing name made more sense when they lived in the general `verify.rs` module, but less so now.

WDYT?

Reply:
I agree that would be better -- as is, the names feel like what is sometimes described as stuttering (also, these names are already pretty long).